OnlineStore/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
please read about proper content of gitignore file and place it to the root folder